package.json

```diff
@@ -1470,7 +1470,7 @@
                 {
                     "command": "aws.ecs.runCommandInContainer",
                     "group": "0@1",
-                    "when": "view == aws.explorer && viewItem == awsEcsContainerNode"
+                    "when": "view == aws.explorer && viewItem =~ /^(awsEcsContainerNodeExecEnabled|awsEcsContainerNodeExecDisabled)$/"
```

Comment:
I believe this statement can be condensed to `/^(awsEcsContainerNodeExec)(.*)$/` . No worries though, we aren't super great about doing this throughout package.json anyway.

Reply:
Once C9 adds `setContext` we can clean these clauses up.

Reply:
Why?
* viewItem is supported and works fine in both IDEs
* There's no "right-click tree node" event hook so we can't evaluate context dynamically

Reply:
I was talking more about some of the other clauses, not so much this one (though it's possible to use it for this one). Things like `copyArn` and `copyName` which can be handled in a cross-cutting way. As soon as we start composing contexts, `setContext` becomes desirable.